File: programs-basic-java/src/test/java/com/local/numbertypes/SpyNumberTest.java
package com.local.numbertypes;

import static com.local.numbertypes.NumberType.SPY_NUMBER;

import java.util.List;

import org.junit.jupiter.api.Test;

class SpyNumberTest extends AbstractNumberTest {

	@Test
	void testSpyNumbers() {
		testNumbersTypeByFile(SPY_NUMBER, "spynumbers.txt");
	}

	@Test
	void testSpyNumbersByList() {
		// Pass the order as 5
		testNumbersTypeByList(SPY_NUMBER, List.of(132,1123,123123), 1);
	}
}

File: programs-basic-kotlin/src/test/kotlin/com/local/numbertypes/SpyNumberTest.kt
package com.local.numbertypes
import org.junit.jupiter.api.Test
class SpyNumberTest : AbstractNumberTest() {
	@Test
	fun testSpyNumbers() {
		testNumbersTypeByFile(com.local.numbertypes.NumberType.SPY_NUMBER, "spynumbers.txt")
	}

	@Test
	fun testSpyNumbersByList() {
		// Pass the order as 5
		testNumbersTypeByList(com.local.numbertypes.NumberType.SPY_NUMBER, java.util.List.of<Int>(132, 1123, 123123), 1)
	}
}